### ISSUE TITLE
data_gen_val didn't used in "train_frcnn.py"

### ISSUE THREAD

**Issue by ntut108318099** [kentaroy47/frcnn-from-scratch-with-keras#84]
Sorry,I found "data_gen_val"  in  "train_frcnn.py" were used once,and it didn't be used during training.

Also, **pascal_voc_parser.py** extracts the test set as well but it has not been used for training. How can I show the validation loss and also the test loss during training?

Apologies if I got something wrong, beginner, here. Thanks for the project!

**Comment by TiasM** [comment]
Hi, sorry i didn't want to make a new thread, but this part of the code:
# first 3 epoch is warmup
	if epoch_num == 3 and options.rpn_weight_path is not None:

the == 3 part i think suppose to be <=3

**Comment by kentaroy47** [comment]
Thanks for the mentioned, let me fix this.

**Comment by TiasM** [comment]
Hi, @kentaroy47 , sorry, i think your previous code was right, i think it should be epoch_num == 3 after all. I already write my thought on the commit section, please have a look. Sorry if my grammar is horrible, Thank You :)

**Comment by kentaroy47** [comment]
@TiasM  thanks for comments! I will look into it and revert if required.

**Comment by kentaroy47** [comment]
I've wrote a comment for the commit and I think the implementation after the commit is right.
I'm going to close this, and please reopen if there are further issues.
